interface/super/manage_site_files.php

```diff
@@ -209,6 +259,28 @@ function msfFileChanged() {
 </center>
 
 </form>
+
+<div id="generate_thumb">
+    <table>
+        <tr>
+            <td class="thumb_title" width="19%">
+                <b><?php echo xl('Generate Thumbnails')?></b>
+            </td>
+            <td class="thumb_subtitle" width="28%">
+                <span><?php echo xl('The thumbnails will be store ' . $storage_method) ?></span>
+            </td>
+            <td class="thumb_msg" width="49%">
+                <span><?php echo $thumbnail_msg ?></span>
+            </td>
+            <td  class="thumb_form" width="16%" style="border-right:none">
+                <form method='post' action='manage_site_files.php#generate_thumb'>
+                    <input style="margin-top: 10px" type="submit" name="generate_thumbnails" value="<?php echo htmlspecialchars(xl('Generate')) ?>">
```

Comment:
use this instead:
<input style="margin-top: 10px" type="submit" name="generate_thumbnails" value="<?php echo xla('Generate') ?>">